ui/language/view.py

```diff
@@ -0,0 +1,48 @@
+from ..braille import from_ascii, format_title, to_ueb_number
+from ..i18n import I18n
+
+i18n = I18n()
+
+def render_help_menu(width, height, page):
+    data = (
+        from_ascii(i18n._('Select a language by using the side')),
+        from_ascii(i18n._('number buttons and pressing forward.')),
+    )
+
+    data = [from_ascii(line) for line in data]
+
+    while len(data) < height:
+        data.append(tuple())
+
+    return tuple(data)
```

Comment:
You are going from `tuple -> list -> tuple` within this function and calling `from_ascii` twice on each line. That can't be right.

Reply:
What do you suggest, here?  I'm not an experienced Python dev.  I'll see what I can do to reduce the steps.